pytket/extensions/pennylane/_metadata.py

```diff
@@ -1,2 +1,2 @@
-__extension_version__ = "0.1.0"
+__extension_version__ = "0.1.1"
```

Comment:
I think the version needs updating here as well: https://github.com/kimaranaicker/pytket-pennylane/blob/main/pytket/extensions/pennylane/pytket_device.py#L24

Could we use a pytket._version import or something similar to do that?

Reply:
I've updated the version manually in pytket_device.py.
I'm being stupid and don't understand what you mean by pytket._version import.
We could open the metadata and read off the extension_version?